src/main/java/github/scarsz/discordsrv/listeners/DiscordAccountLinkListener.java

```diff
@@ -45,8 +48,28 @@ public void onPrivateMessageReceived(PrivateMessageReceivedEvent event) {
 
         DiscordSRV.api.callEvent(new DiscordPrivateMessageReceivedEvent(event));
 
+        // don't link accounts if config option is disabled
+        if (!DiscordSRV.config().getBoolean("MinecraftDiscordAccountEnablePM")) return;
+
+        String reply = DiscordSRV.getPlugin().getAccountLinkManager().process(event.getMessage().getContentRaw(), event.getAuthor().getId());
+        if (reply != null) event.getMessage().reply(reply).queue();
+    }
+
+    @Override
+    public void onGuildMessageReceived(GuildMessageReceivedEvent event) {
+        // don't process messages sent by the bot
+        if (event.getAuthor().getId().equals(event.getJDA().getSelfUser().getId())) return;
+
+        DiscordSRV.api.callEvent(new DiscordGuildMessageReceivedEvent(event));
```

Comment:
This event is already published by the chat listener, it shouldn't be published twice

Reply:
Okay, I removed it